File: app/src/main/java/com/example/minilist/WatchActivity.java
package com.example.minilist;

import android.os.Bundle;
import android.preference.PreferenceManager;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.example.minilist.databinding.ActivityWatchBinding;

public class WatchActivity extends AppCompatActivity {

    private ActivityWatchBinding binding;
    private WatchPresenter presenter;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityWatchBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());
        presenter = new WatchPresenter(this, PreferenceManager.getDefaultSharedPreferences(this));
    }

    public void setText(String text) {
        binding.lineList.setText(text);
    }

    public void setTextColor(int color) {
        binding.lineList.setTextColor(color);
    }

    public void setTextSize(int size, float dimension) {
        binding.lineList.setTextSize(size, dimension);
    }
}

File: app/src/main/java/com/example/minilist/watch/WatchActivity.kt
package com.example.minilist.watch

import android.os.Bundle
import android.preference.PreferenceManager
import androidx.appcompat.app.AppCompatActivity
import com.example.minilist.databinding.ActivityWatchBinding
import java.io.File

class WatchActivity : AppCompatActivity(), WatchWireframe {

    private lateinit var binding: ActivityWatchBinding
    private lateinit var presenter: WatchPresenter

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityWatchBinding.inflate(layoutInflater)
        setContentView(binding.root)
        presenter = WatchPresenter(this, PreferenceManager.getDefaultSharedPreferences(this))
    }

    override fun setText(text: String) {
        binding.lineList.text = text
    }

    override fun setTextColor(color: Int) {
        binding.lineList.setTextColor(resources.getColor(color))
    }

    override fun setTextSize(size: Int, dimension: Int) {
        binding.lineList.setTextSize(size, resources.getDimensionPixelSize(dimension).toFloat())
    }

    override fun getDir(): File = filesDir


}